File: src/main/java/com/server/ordering/domain/dto/response/OrderFoodDto.java
package com.server.ordering.domain.dto.response;

import com.server.ordering.domain.OrderFood;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import static lombok.AccessLevel.PROTECTED;

@Getter @Setter
@AllArgsConstructor
@NoArgsConstructor(access = PROTECTED)
public class OrderFoodDto {

    private String foodName;
    private int price;
    private int count;

    public OrderFoodDto(OrderFood orderFood) {
        this.foodName = orderFood.getFood().getFoodName();
        this.price = orderFood.getPrice();
        this.count = orderFood.getCount();
    }
}

File: src/main/kotlin/com/byeoru/ordering_server/domain/dto/response/OrderFoodDto.kt
package com.byeoru.ordering_server.domain.dto.response

import com.byeoru.ordering_server.domain.OrderFood

class OrderFoodDto(orderFood: OrderFood) {

    val foodName: String
    val price: Int
    val count: Int

    init {
        foodName = orderFood.food.foodName
        price = orderFood.price
        count = orderFood.count
    }
}